presto-clickhouse/src/main/java/io/prestosql/plugin/clickhouse/ClickHouseTableHandle.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.clickhouse;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Joiner;
+import io.prestosql.spi.connector.ColumnHandle;
+import io.prestosql.spi.connector.ConnectorTableHandle;
+import io.prestosql.spi.connector.SchemaTableName;
+import io.prestosql.spi.predicate.TupleDomain;
+
+import javax.annotation.Nullable;
+
+import java.util.Objects;
+import java.util.OptionalLong;
+
+import static java.util.Objects.requireNonNull;
+
+public class ClickHouseTableHandle
```

Comment:
Why can't we use `JdbcTableHandle` ?

Reply:
I have no idea about `JdbcTableHandle' when I wrote the class, I will try to use it